plugins/aws/cloudformation/plainTextParameters.js

```diff
@@ -0,0 +1,77 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'CloudFormation Plaintext Parameters',
+    category: 'CloudFormation',
+    description: 'Ensures CloudFormation parameters that reference sensitive values are configured to use NoEcho.',
+    more_info: 'CloudFormation supports the NoEcho property for sensitive values, which should be used to ensure secrets are not exposed in the CloudFormation UI and APIs.',
+    link: 'https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html',
+    recommended_action: 'Update the sensitive parameters to use the NoEcho property.',
+    apis: ['CloudFormation:describeStacks'],
+    settings: {
+        plainTextParameters: {
+            secretWords: [
+                'secret', 'password', 'privatekey'
+            ]
+        }
```

Comment:
This is a good start, but see here for an example of the format:
https://github.com/cloudsploit/scans/blob/master/plugins/aws/acm/acmCertificateExpiry.js#L12-L18

For example, this section should be:
```suggestion
        plain_text_parameters: {
            name: "CloudFormation Plaintext Parameters",
            description: "A comma-delimited list of parameter strings that indicate a sensitive value",
            regex: "[a-zA-Z0-9,]",
            default: "secret,password,privatekey"
        }
```

Then later, you can create an array by doing a `split(',')` on the words